### ISSUE TITLE
Version numbers resolve to components, but not the search items in the `search` components

### ISSUE THREAD

**Issue by fschiettecatte** [ga4gh-discovery/ga4gh-case-discovery#11]
Version numbers resolve to components, but not the search items in the `search` components, do we need those to be versioned as well? For example:

```
{
    "meta": {
        "request": {
            "components": {
                "search": {
                    "gene": "1.0.0",
                    "type": "1.0.0",
                    "feature": "1.0.0"
                },
            }
        }
    }
}
```

**Comment by Relequestual** [comment]
My intention was for the root object key `components`'s value to be the search components, and for any metadata about the search request to be in `meta.components`.

In my proposal:

> Metadata may contain any sort of information about the actual data. This might include a URI which has further service description information encoded in JSON, but that's out of the scope or this proposal.

This would include the `queryIdentification` component for example, so `meta.components.queryIdentification` as an object.

I didn't want to get into defining specific components due to lack of time, but was more focused on the structure.

I think I need to open a new issue to reflect this missunderstanding, which needs to be resolve before this one.

**Comment by Relequestual** [comment]
Blocked by #25

**Comment by Relequestual** [comment]
Closing as #25 is resolved.
